examples/auctions/example_response_full.json

```diff
@@ -0,0 +1,198 @@
+{
+  "status": 200,
+  "data": {
```

Comment:
Don't mean to be picky but it would be cool if summations were added: total auctions, auctions sold, coins to pickup, value if all sold